ps_socialfollow.php

```diff
@@ -58,7 +58,7 @@ public function __construct()
         $this->displayName = $this->trans('Social media follow links', array(), 'Modules.Socialfollow.Admin');
         $this->description = $this->trans('Allows you to add information about your brand\'s social networking accounts.', array(), 'Modules.Socialfollow.Admin');
 
-        $this->ps_versions_compliancy = array('min' => '1.7.1.0', 'max' => _PS_VERSION_);
+        $this->ps_versions_compliancy = array('min' => '1.7.4.0', 'max' => _PS_VERSION_);
```

Comment:
@matks This change of version compliance is not related to my changes.
Even before, this module required PHP 7, because it used syntax that is invalid in PHP 5.6.

Reply:
Yes, I added this change. If it was marked as compatible with PS 1.7.1 before, it was a mistake. Let's seize the opportunity to fix that 😄